src/plugins/unified_histogram/public/chart/histogram.tsx

```diff
@@ -197,7 +185,7 @@ export function Histogram({
     }
 
     & .lnsExpressionRenderer {
-      width: ${chartSize};
+      width: ${attributes.visualizationType === 'lnsMetric' ? '90$' : '100%'};
```

Comment:
Oops, looks like the latest changes switched the width from `90%` to `90$`. I opened a PR to fix it here and promise not to charge 90$ for it: #188150.